### ISSUE TITLE
Expose Token and TokenAndSpan?

### ISSUE THREAD

**Issue by dsherret** [swc-project/swc#509]
Should `Token`, `TokenAndSpan`, and related types be exposed?

I'm testing out getting tokens by doing:

```rust
let capturing = Capturing::new(lexer);
let mut parser = Parser::new_from(session, capturing);
let parse_module_result = parser.parse_module();

let tokens = parser.input().take();
println!("Tokens: {:?}", tokens);

// not much I can do with tokens here
```

...and I see the [result of that `take()` is `TokenAndSpan`](https://swc-project.github.io/rustdoc/swc_ecma_parser/struct.Capturing.html#method.take), but I'm not able to use `Token` or `TokenAndSpan` as they seem to be in the private `token` module.

Thanks!

**Comment by kdy1** [comment]
@dsherret published `swc_ecma_ast@0.10.1` and `swc_ecma_parser@0.12.3`

**Comment by dsherret** [comment]
Awesome. Thanks @kdy1!
